Gemfile.lock

```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    github_changelog_generator (1.8.1)
+    github_changelog_generator (1.9.0)
```

Comment:
I will update version with release (it will bumped automatically)
please, revert this bump